apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.StatisticsTests.cs

```diff
@@ -22,11 +24,13 @@ public partial class SessionPoolTests
     {
         public sealed class StatisticsTests
         {
-            [Fact]
+            [Fact]
+            [Obsolete]

```

Comment:
We don't mark tests as obsolete, we use the the ignore warning directive on the piece of code that uses the deprecated code.

And also not even sure you need to mark this one as you have changed all the code to use the new class.

But, it'd still be good to maybe continue to tests the old class until we have removed it.